apis/fluent/atrium-api-fluent/api/main/atrium-api-fluent.api

```diff
@@ -518,6 +518,13 @@ public final class ch/tutteli/atrium/api/fluent/en_GB/SequenceSubjectChangersKt
 	public static final fun asList (Lch/tutteli/atrium/creating/Expect;Lkotlin/jvm/functions/Function1;)Lch/tutteli/atrium/creating/Expect;
 }
 
+public final class ch/tutteli/atrium/api/fluent/en_GB/SqlExceptionFeatureExtractorsKt {
```

Comment:
you also need to generate the api for Kotlin 1.9 and newer. Run `KOTLIN_VERSION=2.0 ./gradlew apiDump` to achieve this